scrapers/scrape_be_tests.py

```diff
@@ -23,7 +31,9 @@
 
             if sc.find(r'^(Kalender.*)', headers[col_num]) is not None:
                 td.week = value
-                td.year = '2020'
+                if( int(td.week) == 53 ):
```

Comment:
the parenthesis are not needed here - `if int(td.week) == 53:` is sufficient :sweat_smile:

Reply:
🙈